packages/medusa-js/src/types/product-variant/index.ts

```diff
@@ -49,3 +50,12 @@ export interface Currency {
   symbol_native: string;
   name: string;
 }
+
```

Comment:
I agree 100% with this and I actually think that we should change the API to allow this instead:
`?id[]=id1&id[]=id2`

Thereby avoiding this situation